### PR TITLE
Initial API + Python package

### PR DESCRIPTION
**Pull request by Krigpl**

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from setuptools import setup
+
+setup(
+    name='Golem-Task-App',
```
Comment (Wiezzel) [inline python/setup.py]:
```suggestion
    name='Golem-Task-Api',
```

File: proto/golem_task_api/golem_task_api.proto [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+message VerifyReply {
+  bool success = 1;
+}
+
+message RunBenchmarkRequest {
+}
+
+message RunBenchmarkReply {
```
Comment (Wiezzel) [inline proto/golem_task_api/golem_task_api.proto]: Why empty? We need benchmark score.

File: python/golem_task_api/constants.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import sys
+
+from . import constants_pb2
+
+for field, value in constants_pb2.DESCRIPTOR.GetOptions().ListFields():
+    setattr(sys.modules[__name__], field.name, value)
```
Comment (Wiezzel) [inline python/golem_task_api/constants.py]: Ugh. :nauseated_face:

File: proto/golem_task_api/golem_task_api.proto (hunk 2 above)
Comment (Krigpl) [reply proto/golem_task_api/golem_task_api.proto]: Right

File: python/golem_task_api/constants.py (hunk 3 above)
Comment (Krigpl) [reply python/golem_task_api/constants.py]: Yeah

**Pull request merged**